### ISSUE TITLE
Missing `eap` field for `productDescriptor`

### ISSUE THREAD

**Issue by missingdays** [JetBrains/intellij-platform-gradle-plugin#1703]
### What happened?

`productDescriptor` builder is missing `eap` flag

### Steps to reproduce

```
intellijPlatform {
    pluginConfiguration {
          productDescriptor {
              eap = true // doesn't compile
          }
        }
    }
```

### Gradle IntelliJ Plugin version

2.0.0-beta8

### Gradle version

8.7

### Operating System

None

**Comment by hsz** [comment]
How did you come with the `eap` property? It's not available in the API: https://plugins.jetbrains.com/docs/intellij/tools-intellij-platform-gradle-plugin-extension.html#intellijPlatform-pluginConfiguration-productDescriptor

PS, please upgrade to the latest `2.0.0-RC1`

**Comment by hsz** [comment]
This flag is now available in `2.0.0-RC1`
